docs/intro.asciidoc

```diff
@@ -6,17 +6,23 @@ endif::[]
 [[intro]]
 == Introduction
 
-Welcome to the APM .NET Agent documentation.
+The Elastic APM .NET Agent automatically measures the performance of your application and tracks errors.
+It has built-in support for the most popular frameworks and routers,
+as well as a simple API which allows you to instrument any application.
 
-Elastic APM automatically measures the performance of your application and tracks errors.
-For example, it records spans for database queries and transactions for incoming HTTP requests.
-See <<supported-technologies>> to learn about which frameworks and technologies are supported by Elastic APM .NET agent.
+[float]
+[[how-it-works]]
+=== How does the Agent work?
+
+The agent auto-instruments <<supported-technologies,supported technologies>> and records interesting events,
+like HTTP requests and database queries. To do this, it {CONTENT_HERE}
+This means that for the supported technologies, there are no code changes required.
```

Comment:
Sooo... the `no code changes required` is not 100% accurate: there is a 1 liner (or you need to register the IIS module) in order to to turn on these auto instrumentations. Maybe we could say, `there are no code changes required, other than enabling auto-instrumentation as shown on the [setup page](LinkToSetupPage)`. With this we'd be on the safe side and not create the impression that adding the NuGet packages is enough.

Reply:
Great idea, thank you!